src/chordsymbol.ts

```diff
@@ -680,6 +707,9 @@ export class ChordSymbol extends Modifier {
     ctx.openGroup(classString, this.getAttribute('id'));
 
     const start = note.getModifierStartXY(Modifier.Position.ABOVE, this.index);
+    // TEMPORARILY DISABLE. Will be fixed in RONYEH's FONTS PR.
```

Comment:
I had to add some of these eslint-disable / ts-ignore to this PR. They will be removed once the Fonts branch is merged.